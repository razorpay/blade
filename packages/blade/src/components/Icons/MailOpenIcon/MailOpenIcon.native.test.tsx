import MailOpenIcon from '.';
import renderWithTheme from '~utils/testing/renderWithTheme.native';

describe('<MailOpenIcon />', () => {
  it('should render MailOpenIcon', () => {
    const renderTree = renderWithTheme(
      <MailOpenIcon color="feedback.icon.neutral.intense" size="large" />,
    ).toJSON();
    expect(renderTree).toMatchSnapshot();
  });
});
