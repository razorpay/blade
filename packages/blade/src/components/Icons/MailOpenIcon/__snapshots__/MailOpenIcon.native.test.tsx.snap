// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MailOpenIcon /> should render MailOpenIcon 1`] = `
<View
  style={
    {
      "flex": 1,
    }
  }
>
  <RNSVGSvgView
    accessibilityElementsHidden={true}
    align="xMidYMid"
    bbHeight="20px"
    bbWidth="20px"
    data-blade-component="icon"
    fill="none"
    focusable={false}
    height="20px"
    importantForAccessibility="no-hide-descendants"
    meetOrSlice={0}
    minX={0}
    minY={0}
    style={
      [
        {
          "backgroundColor": "transparent",
          "borderWidth": 0,
        },
        [
          {},
        ],
        {
          "flex": 0,
          "height": 20,
          "width": 20,
        },
      ]
    }
    vbHeight={24}
    vbWidth={24}
    width="20px"
  >
    <RNSVGGroup
      fill={null}
      propList={
        [
          "fill",
        ]
      }
    >
      <RNSVGPath
        clipRule={0}
        d="M22.7829 8.36084C22.6408 8.11932 22.4368 7.92021 22.1918 7.7841L19.4643 6.26881V3.75C19.4643 3.02065 19.1746 2.32118 18.6588 1.80546C18.1431 1.28973 17.4436 1 16.7143 1H7.28571C6.55637 1 5.8569 1.28973 5.34117 1.80546C4.82544 2.32118 4.53571 3.02065 4.53571 3.75V6.26881L1.80824 7.78408C1.56328 7.92018 1.35917 8.11931 1.21706 8.36084C1.07495 8.60236 1.00001 8.8775 1 9.15773V19.0714C1 20.1133 1.4139 21.1126 2.15065 21.8493C2.8874 22.5861 3.88665 23 4.92857 23H19.0714C20.1133 23 21.1126 22.5861 21.8493 21.8493C22.5861 21.1126 23 20.1133 23 19.0714V9.15773C23 8.87749 22.925 8.60236 22.7829 8.36084ZM4.92857 7.84824V7.84823L2.57142 9.15773L2.57143 19.0714V9.15773L4.92857 7.84824ZM4.53571 8.51589V10.4482C4.37829 10.4162 4.21392 10.4229 4.05726 10.47C3.80784 10.5449 3.59835 10.7157 3.47483 10.945C3.35131 11.1742 3.32386 11.4431 3.39851 11.6926C3.47316 11.9421 3.64421 12.152 3.87336 12.2757L11.5348 16.4011C11.6779 16.4777 11.8377 16.5179 12 16.5179C12.1623 16.5179 12.3228 16.4774 12.4659 16.4007L20.127 12.2755C20.3562 12.1517 20.5268 11.9421 20.6015 11.6926C20.6761 11.4431 20.6487 11.1742 20.5252 10.945C20.4016 10.7157 20.1922 10.5449 19.9427 10.47C19.7861 10.4229 19.6217 10.4162 19.4643 10.4482V8.51589L21.0357 9.38889V19.071C21.0351 19.5919 20.8279 20.0913 20.4596 20.4596C20.0913 20.8279 19.5919 21.0351 19.071 21.0357H4.92902C4.40813 21.0351 3.90873 20.8279 3.5404 20.4596C3.17211 20.0913 2.96492 19.5918 2.96429 19.071L2.96429 9.38889L4.53571 8.51589ZM6.73053 3.19481C6.87788 3.04746 7.07779 2.96456 7.28616 2.96429H16.7138C16.9222 2.96456 17.1221 3.04746 17.2695 3.19481C17.4169 3.3422 17.4998 3.54202 17.5 3.75045V11.4584L12 14.4202L6.5 11.4584V3.75018C6.50031 3.54185 6.58321 3.34213 6.73053 3.19481ZM7.28571 2.57143H16.7143L7.28571 2.57142C6.97324 2.57178 6.67368 2.69607 6.45273 2.91702C6.23178 3.13796 6.1075 3.43753 6.10714 3.75L6.10714 11.6931V3.75C6.1075 3.43753 6.23179 3.13797 6.45273 2.91702C6.67368 2.69607 6.97325 2.57179 7.28571 2.57143Z"
        fill={
          {
            "payload": 4280563015,
            "type": 0,
          }
        }
        fillRule={0}
        propList={
          [
            "fill",
            "fillRule",
          ]
        }
      />
    </RNSVGGroup>
  </RNSVGSvgView>
</View>
`;
